opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/container-organization-sharing-test.ts

```diff
@@ -0,0 +1,200 @@
+    expect(execution).not.toBeNull();
+    expect(execution.status).toEqual('success');
+  }, FIVE_MINUTES);
+  // Python lib is fixed but we need to wait for a new release
+  it('Should import update succeed', async () => {
+    const execution = await execChildPython(testContext, ADMIN_USER, PYTHON_PATH, 'local_importer.py', importOpts);
+    expect(execution).not.toBeNull();
+    expect(execution.status).toEqual('success');
+  }, FIVE_MINUTES);
+});
+
+describe('Organization sharing standard behavior for container', () => {
+  let reportInternalId: string;
+  let organizationId: string;
+  let settingsInternalId: string;
+  let platformOrganizationId: string;
+  it('should load Report', async () => {
+    const report = await findById(testContext, ADMIN_USER, 'report--57162a65-2a58-560b-9a65-47c3f040f3d4'); // Report is in DATA-TEST-STIX_v2.json
+    reportInternalId = report.internal_id;
+  });
+  it('should plateform organization sharing and EE activated', async () => { // TODO extract set/unset EE and orga platfor in testQueryHelpers
+    // Get organization id
+    platformOrganizationId = await getOrganizationIdByName(PLATFORM_ORGANIZATION.name);
+
+    // Get settings ID
+    const SETTINGS_READ_QUERY = gql`
+      query settings {
+        settings {
+          id
+          platform_organization {
+            id
+            name
+          }
+        }
+      }
+    `;
+    const queryResult = await adminQuery({ query: SETTINGS_READ_QUERY, variables: {} });
+    settingsInternalId = queryResult.data?.settings?.id;
+
+    // Set plateform organization
+    const platformOrganization = await adminQueryWithSuccess({
+      query: PLATFORM_ORGANIZATION_QUERY,
+      variables: {
+        id: settingsInternalId,
+        input: [
+          { key: 'platform_organization', value: platformOrganizationId },
+          { key: 'enterprise_edition', value: new Date().getTime() },
+        ]
+      }
+    });
+    expect(platformOrganization?.data?.settingsEdit.fieldPatch.platform_organization).not.toBeUndefined();
+    expect(platformOrganization?.data?.settingsEdit.fieldPatch.enterprise_edition).not.toBeUndefined();
+    expect(platformOrganization?.data?.settingsEdit.fieldPatch.platform_organization.name).toEqual(PLATFORM_ORGANIZATION.name);
+  });
+  it('should share Report with Organization', async () => {
+    // Get organization id
+    organizationId = await getOrganizationIdByName(TEST_ORGANIZATION.name);
+    const organizationSharingQueryResult = await adminQueryWithSuccess({
+      query: ORGANIZATION_SHARING_QUERY,
+      variables: { id: reportInternalId, organizationId }
+    });
+    expect(organizationSharingQueryResult?.data?.stixCoreObjectEdit.restrictionOrganizationAdd).not.toBeNull();
+    expect(organizationSharingQueryResult?.data?.stixCoreObjectEdit.restrictionOrganizationAdd.objectOrganization[0].name).toEqual(TEST_ORGANIZATION.name);
+
+    // Need background task magic to happens for sharing
+    await taskHandler();
```

Comment:
This is a kind of hack, not sure of this. But we will have some random issue if the manager has not run before next test. The alternative would be to do this test at domain level in taskManager-test, calling taskManager.executeShare.